stdlib/multiprocessing/context.pyi

```diff
@@ -53,22 +57,50 @@ class BaseContext(object):
         initargs: Iterable[Any] = ...,
         maxtasksperchild: Optional[int] = ...,
     ) -> multiprocessing.pool.Pool: ...
-    # TODO: typecode_or_type param is a ctype with a base class of _SimpleCData or array.typecode Need to figure out
-    # how to handle the ctype
-    # TODO: change return to RawValue once a stub exists in multiprocessing.sharedctypes
-    def RawValue(self, typecode_or_type: Any, *args: Any) -> Any: ...
-    # TODO: typecode_or_type param is a ctype with a base class of _SimpleCData or array.typecode Need to figure out
-    # how to handle the ctype
-    # TODO: change return to RawArray once a stub exists in multiprocessing.sharedctypes
-    def RawArray(self, typecode_or_type: Any, size_or_initializer: Union[int, Sequence[Any]]) -> Any: ...
-    # TODO: typecode_or_type param is a ctype with a base class of _SimpleCData or array.typecode Need to figure out
-    # how to handle the ctype
-    def Value(self, typecode_or_type: Any, *args: Any, lock: bool = ...) -> sharedctypes._Value: ...
-    # TODO: typecode_or_type param is a ctype with a base class of _SimpleCData or array.typecode Need to figure out
-    # how to handle the ctype
+    @overload
+    def RawValue(self, typecode_or_type: Type[_CT], *args: Any) -> _CT: ...
+    @overload
+    def RawValue(self, typecode_or_type: Union[str, Type[_CData]], *args: Any) -> Any: ...
```

Comment:
`Type[_CData]` is already covered by the first overload and will never match in the second one. Same for `RawArraw` and the corresponding definitions in `.sharedctypes`.

Reply:
_CData is covered but the union is not - unless I'm missing something. So If I'm giving this Union[str, _CArray] I want this to return Any. Right?

Reply:
Type checkers should be able to build a union themselves. For example with mypy:

```python
from typing import overload, Union

@overload
def foo(x: str) -> str: ...
@overload
def foo(x: int) -> int: ...

x: Union[int, str]
reveal_type(foo(x))  # Revealed type is "Union[builtins.int, builtins.str]"
```